tasks/migrate-certificates.ts

```diff
@@ -417,7 +417,6 @@ export const GET_MIGRATE_CERTIFICATES_TASK = () =>
         `Certificate contract address: ${certificateContract.address}`
       );
       logger.info(`Signer address: ${signerAddress}`);
-      // const fireblocksSigner = removalContract.signer as FireblocksSigner;
```

Comment:
curious why this was needed in the past-- any idea?

Reply:
I don't think it ever was.  I'm sure there was an original intention for it but I remember that line living commented in the scripts for their lifetime